<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actuatortest_Dialog</class>
 <widget class="QDialog" name="actuatortest_Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>397</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>397</width>
    <height>142</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Actuator Test</string>
  </property>
  <widget class="QLabel" name="actuatortitle_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>49</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Actuator:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statustitle_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>79</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="actuator_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>49</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>79</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_pushButton">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>106</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Actuator Test:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
